meson.build

```diff
@@ -380,7 +380,7 @@ endforeach
 sed = find_program('sed')
 grep = find_program('grep')
 awk = find_program('awk')
-m4 = find_program('m4')
+m4 = find_program('/usr/bin/m4')
```

Comment:
Now that 0.40.0 has been released, I would probably revert this commit and bump the minimum required version of meson to 0.40.0.